src/Wallabag/CoreBundle/Helper/RuleBasedTagger.php

```diff
@@ -36,6 +38,11 @@ public function tag(Entry $entry)
                 continue;
             }
 
+            $this->logger->info('Matching rule.', [
+                'rule' => $rule->getRule(),
+                'tags' => $rule->getTags(),
```

Comment:
The object is `$rule` and `Rule` and `Tags` are fields in wallabag_tagging_rule table.